api/ai/new/priority_algorithm/mutations/local_max.py

```diff
@@ -1,9 +1,10 @@
 import itertools
+import random
 from typing import List, Dict
 
-from api.ai.new.priority_algorithm.priority.interfaces import Priority
-from api.ai.new.priority_algorithm.mutations.utils import score
-from api.ai.new.priority_algorithm.custom_models import PriorityTeamSet
+from api.ai.priority_algorithm.interfaces import Priority
```

Comment:
wait be careful with these imports, the priority algorithm outside of the `/new` folder is going to be deleted

Reply:
Oh right